src/App.tsx

```diff
@@ -20,20 +26,64 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 3,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1000,
+    infinite: false,
+  };
+
+  handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState(state => ({
+      ...state,
+      [event.target.name]: +(event.target.value),
```

Comment:
nice, but let's convert data types explicitly
```suggestion
      [event.target.name]: Number(event.target.value),
```

Reply:
Thank you Roman.  I added it now.